haas/test_common.py

```diff
@@ -127,6 +127,7 @@ def newDB():
 def releaseDB():
     """Do we need to do anything here to release resources?"""
     with app.app_context():
+        db.reflect()
```

Comment:
weird thing I found on the internet. Have no idea what it does. Introduced in commit of the name "I have literally no idea why this worked". The comment right above it describes the problem I was having that it fixed (somehow)